sympy/utilities/lambdify.py

```diff
@@ -797,10 +806,19 @@ def _lambdifygenerated(x):
                            'allow_unknown_functions': True,
                            'user_functions': user_functions})
 
+
+    if  type(args) is set:
+        warnings.warn('WARNING: The list of arguments is a `set`. The outcome of enumerate(args) is used to `lambdify`, but its outcome is not predictable. Lambdify will stop here and return `False`', category=DeprecationWarning)
+        return False  # Anything else that block the code would be fine
+
+
+
```

Comment:
Blocks of code within this function are separated by one blank line. You have in some cases used two blank lines and here there are three.